src/js/background/background.js

```diff
@@ -367,6 +393,14 @@ browser.runtime.onMessage.addListener(async (m, sender, _sendResponse) => {
     case "iframeCloseRelayInPageMenu":
       browser.tabs.sendMessage(sender.tab.id, {message: "iframeCloseRelayInPageMenu"});
       break;
+    case "fetchRequestFromBackground": 
+      response = await fetchRequestFromBackground(
+        m.url, 
+        m.fetchMethod,
+        m.body,
+        m.opts
```

Comment:
Question: I didn't run into any issues but is there any danger passing null/undefined values here? (Most of the time when this message is called, `fetchMethod`, `body` and `opts` are blank.

Reply:
I wouldn't think so, but your asking the question makes me nervous :P What danger were you thinking about?

Reply:
Rewrote this function so that if the URL param is not included, it fails. Everything else has a defined "default". 

https://github.com/mozilla/fx-private-relay-add-on/blob/mv3-spike-followup-fetch-migration/src/js/background/background.js#L40-L44